ibm_was/datadog_checks/ibm_was/data/conf.yaml.example

```diff
@@ -0,0 +1,60 @@
+init_config:
+
+instances:
+  ## @param servlet_url - string - required
+  ## The URL of the PerfServlet configured to report metrics
+  #
+  - servlet_url: http://localhost/wasPerfTool/servlet/perfservlet
+
+  ## @param skip_proxy - boolean - optional - default: False
+  ## Wheter or not to skip the proxy configured in the Agent's config file when
+  ## making the request to the PerfServlet 
+  #
+  # skip_proxy: False
+
+  ## @param collect_jdbc_stats - boolean - optional - default: True
+  ## Whether or not to collect JDBC Connection Pool stats
+  #
+  # collect_jdbc_stats: True
+
+  ## @param collect_jvm_stats - boolean - optional - default: True
+  ## Whether or not to collect JVM Runtime stats
+  #
+  # collect_jvm_stats: True
```

Comment:
Why not always collect this?

Reply:
Its possible that a user will be collecting the overall JVM health metrics through something like JMX. For that reason, and that in code each collection_<>_stats is dynamically set, I've treated them all equally and allowed them to be enabled/disabled, with enabled being the default.